website/landing/components/Community/Community.tsx

```diff
@@ -0,0 +1,187 @@
+import { styled } from "../../stitches.config";
+import { Box, List, Text } from "../common";
+
+const COMMUNITIES_ICONS = {
```

Comment:
The project doesn't have reusable icons (at least yet), so I didn't go the extra mile of setting up an icon library.